### ISSUE TITLE
A big THANK YOU for this repo

### ISSUE THREAD

**Issue by xD0135** [brettscott/cloudflare-worker-tunnel-mysql-example#2]
Not an issue, I just wanted to say THANK YOU for putting together a very well detailed step-by-step instructions on how to get Workers to work with a self-hosted database. Well done sir, well done. 🥇 It helped me get unblocked and get my sanity back :)

**Comment by vivekpatel5996** [comment]
Hi @xD0135,  @brettscott 

We are using Hono framework for Cloudflare worker and have added `deno` and `driver/mysql` both the folders into my project and when I am trying to connect as per this [](https://github.com/brettscott/cloudflare-worker-tunnel-mysql-example/blob/main/src/index.ts)  but it's not working and throwing error like below 

![image](https://github.com/brettscott/cloudflare-worker-tunnel-mysql-example/assets/20946671/005c903c-da03-42d3-a5bc-ab432acc0297)

Your help will be appreciated. Thanks

**Comment by brettscott** [comment]
@vivekpatel5996 please raise a separate issue.  Thanks.

**Comment by brettscott** [comment]
@xD0135 thanks for your kind words 👍
